setup.py

```diff
@@ -37,7 +37,7 @@ def filepath(fname):
     ],
     install_requires=[
         "numpy>=1.6.0",
-        "scipy>=1.0",
+        "scipy<=1.2.1",
```

Comment:
can pip handle lower _and_ upper bounds? I'd still like the requirement that scipy >= 1.0.